test/end-to-end/test.end-to-end.deploy-slack.js

```diff
@@ -116,6 +136,24 @@ describe('End-to-End tests: Slack Deploy UI', () => {
 
         assert.deepEqual(result, expectedResult);
       })
+      .then(() => {
+        return validatePipelineCreation(deploymentName);
+      })
+      .then(action => {
+        assert(
+          true,
+          _.isEqual(action.exec.components, expectedMainSequenceActions)
```

Comment:
i think this assert will never fail. the first argument of assert is the check, and since you set that to true, the assert doesn't really check anything.